lineman/app/components/start_proposal_button/start_proposal_button.coffee

```diff
@@ -0,0 +1,12 @@
+angular.module('loomioApp').directive 'startProposalButton', ->
+  scope: { discussion: '='}
+  restrict: 'E'
+  templateUrl: 'generated/components/start_proposal_button/start_proposal_button.html'
+  replace: true
+  controller: ($scope, Records, ModalService, ProposalForm, CurrentUser, AbilityService) ->
+
+    $scope.canStartProposal = ->
+      AbilityService.canStartProposal $scope.discussion
```

Comment:
What do you think of this AbilityService, @robguthrie?

My discomforts with current permissions:
- Injecting a single instance from the record store as a dependency is a little odd; feels much more natural to have a service
- There's a ton of permissions, and having them on the user model will bloat it quite a bit
- It feels odd to be able to go 'someOtherUser.canCreateProposals()' or something like this; I think we only need to know about permissions for our current user, no?